### DISCUSSION TITLE
Is there are a way to add a footer to Quarto dashboards?

### DISCUSSION THREAD

**Discussion by wvictor14** [quarto-dev/quarto-cli#10393] [Q&A] [answered]
### Description

I could not find any mention of adding footers in Quarto dashboards in the documentation.

I have tried:

- using a `<footer>`  + `<p></p>` tags but these show up as a card, and only on one page in multi-page layouts
- using the `include-after-body` yaml option, but the output is the same as above, it shows up as a card and only on the last page

**Comment by mcanouil** [comment, marked as answer]
That's HTML.
You need to look and understand the structure of the document in order to add things such as "footer".

See for example https://github.com/quarto-dev/quarto-cli/discussions/8194 which you can use/adapt.
